### ISSUE TITLE
Intent to Deprecate

### ISSUE THREAD

**Issue by JRG-Developer** [JRG-Developer/AFNetworking-ImageActivityIndicator#7]
@AnthonyMDev 

I intend to deprecate this library and recommend using <a href="https://github.com/JRG-Developer/UIImageView-ALActivityIndicatorView">UIImageView+ALActivityIndicatorView</a> instead.

`UIImageView-ALActivityIndicatorView` is more generic and can be used with _any_ networking library (not just AFNetworking).

Any problems or concerns about deprecating this?

I've already moved onto using `Alamofire` and `AlamofireImage`, which I believe you have as well, right?

**Comment by AnthonyMDev** [comment]
Yeah that's fine with me! I've actually built my own solution in Swift recently. 

Thanks for the heads up!
